_recipes/creamy-vegan-gnocchi-with-garlic-and-kale.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Creamy Vegan Gnocchi With Garlic and Kale
+category: Dinners
+tags: 
+  - hearty
+  - pasta
+  - vegan
+prepTime:
+  - PT5M
+cookTime:
+  - PT7M
+recipeYield: 3 meals
+recipeCuisine: Italian
+recipeIngredient:
+  - slug: olive oil
```

Comment:
The slugs are used to link to pages about ingredients (which show you what other recipes use a certain ingredient). As such they should be URL slugs,

```suggestion
  - slug: olive-oil
```